app/src/main/java/org/oppia/android/app/translation/AppLanguageResourceHandler.kt

```diff
@@ -157,6 +157,9 @@ class AppLanguageResourceHandler @Inject constructor(
       AudioLanguage.FRENCH_AUDIO_LANGUAGE -> getLocalizedDisplayName("fr")
       AudioLanguage.CHINESE_AUDIO_LANGUAGE -> getLocalizedDisplayName("zh")
       AudioLanguage.BRAZILIAN_PORTUGUESE_LANGUAGE -> getLocalizedDisplayName("pt", "BR")
+      AudioLanguage.ARABIC_LANGUAGE -> getLocalizedDisplayName("ar", "EG")
+      AudioLanguage.NIGERIAN_PIDGIN_LANGUAGE ->
+        resources.getString(R.string.nigerian_pidgin_localized_language_name)
```

Comment:
Correct. Ideally we'll be moving audio languages to OppiaLanguage in general and relying only on our untranslated language names for displaying them. I needed to do it here because otherwise Android doesn't know what to call the language.

Reply:
Got it, thanks.